### PR TITLE
Adding component actions

### PR DESCRIPTION
**Pull request by gabrielreisn**
This PR depends on #2 !!

- includes functionalities "Add todo" and "Remove Todo"

### CODE REVIEW

File: src/components/TodoContainer.js [hunk 1]
```diff
@@ -22,21 +22,63 @@ const textFieldStyle = {
 };
 
 class TodoContainer extends Component {
+  constructor() {
```
Comment (jgcmarins) [inline src/components/TodoContainer.js]: you can avoid constructor and all binds

File: src/components/TodoContainer.js [hunk 2]
```diff
@@ -22,21 +22,63 @@ const textFieldStyle = {
...
+      textField: '',
+      todoText: [],
+    };
+
+    this.handleKeyPress = this.handleKeyPress.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.deleteFunction = this.deleteFunction.bind(this);
+  }
+
+  handleKeyPress(evt) {
```
Comment (jgcmarins) [inline src/components/TodoContainer.js]: you can also use de es6 arrow function sintax `handleKeyPress = (evt) => { ... }`

File: package.json [hunk 3]
```diff
@@ -13,9 +13,27 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "precommit": "lint-staged"
```
Comment (jgcmarins) [inline package.json]: also add `prettier` to scripts and call it on `lint-staged`
example: `"prettier": "prettier --write --single-quote true --trailing-comma all --print-width 120",`

File: package.json [hunk 4]
```diff
@@ -13,9 +13,27 @@
...
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix",
```
Comment (jgcmarins) [inline package.json]: add `yarn prettier` before `eslint --fix`

File: .eslintrc.json [hunk 5]
```diff
@@ -0,0 +1,4 @@
+{
+  "extends": ["standard", "prettier"],
+  "plugins": ["react", "prettier"]
+}
```
Comment (jgcmarins) [inline .eslintrc.json]: use this rules, instead: https://github.com/entria/ReactNavigationRelayModern/blob/master/.eslintrc

**Comment by gabrielreisn** [comment]
Sorry for mixing up things, already changed to (configuring-lint-and-flow) branch

**Comment by jgcmarins** [comment]
Its ok. Just make the changes and merge it \o/

**Pull request merged**